src/ssh/test-sshbridge.c

```diff
@@ -289,26 +290,31 @@ setup (TestCase *tc,
 #if LIBSSH_085
   if (fixture->test_home_ssh_config)
     {
-      gchar *content;
-      gchar **host_port;
+      g_autoptr(GString) content = g_string_new (NULL);
+      g_auto(GStrv) host_port;
```

Comment:
This is safe right now, but by convention please initialize to NULL. This is otherwise a trap if we ever put some `return` between here and assigning host_port.